spec/requests/api/v1/registration_request_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+describe 'Registration API' do
+  it 'can register a new user', :vcr do
+    post '/api/v1/users?email=whatever@example.com&password=password&password_confirmation=password'
```

Comment:
I couldn't figure out how to do a post request with the secure stuff in the headers, but my instinct is that in the real world we wouldn't want to pass them like this.